butte-build/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "butte-build"
+version = "0.1.0"
+authors = ["Phillip Cloud <cpcloud@gmail.com>"]
+description = "A Rust implementation of flatbuffers"
+license = "MIT"
+repository = "https://github.com/flatbutte/butte"
+readme = "README.md"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Nuke this, probaby. Remember to add tags, etc for when this goes on crates.io (see the link)